website/docs/guides/docs/docs-multi-instance.mdx

```diff
@@ -1,50 +1,45 @@
 ---
 id: multi-instance
-description: Use multiple docs plugin instances on a single Docusaurus site.
+description: Use the multi-instance feature to serve 2 or more doc sets.
 slug: /docs-multi-instance
 ---
 
 # Docs Multi-instance
 
-The `@docusaurus/plugin-content-docs` plugin can support [multi-instance](../../using-plugins.mdx#multi-instance-plugins-and-plugin-ids).
+The `@docusaurus/plugin-content-docs` plugin supports [multi-instance](../../using-plugins.mdx#multi-instance-plugins-and-plugin-ids). The multi-instance feature allows a site to serve 2 or more distinct sets of documentation from the same Docusaurus source.
 
 :::note
 
-This feature is only useful for [versioned documentation](./versioning.mdx). It is recommended to be familiar with docs versioning before reading this page. If you just want [multiple sidebars](./sidebar/multiple-sidebars.mdx), you can do so within one plugin.
+If you only want multiple sidebars, use a `sidebar` plugin instead. See [Using multiple sidebars](./sidebar/multiple-sidebars.mdx) for how to do this.
```

Comment:
I still believe that it's mostly useful for versioned docs set. If you have iOS/Android SDKs but they are not versioned, you can just use one plugin instance for both and that's enough. `./docs/ios` + `./docs/android` on a single instance (even if the doc sets are large)

> The Note is misleading. I don't need to use versioning to use this feature. It simply makes sense in many use cases.

Please describe the many use cases

---

What is a "sidebar plugin" you refer to here? This concept does not exist. 

The former docs said you could use multiple sidebars within ONE docs plugin (ie no concept of sidebar plugin)